app/models/integration.rb

```diff
@@ -47,23 +47,15 @@ class Integration < Ekylibre::Record::Base
               converter: proc { |parameters| ActionIntegration::Parameters.cipher(parameters) }
 
   validate do
-    if ciphered_parameters_changed?
-      if integration_type
-        if authentication_mode == :check
-          check_connection attributes do |c|
-            c.redirect do
-              errors.add(:parameters, :check_redirected)
-            end
-            c.error do
-              errors.add(:parameters, :check_errored)
-            end
-          end
-        elsif authentication_mode == :check
-          list = parameters.keys.map(&:to_s)
```

Comment:
This was never reached. (`if a {...} elsif a {...}` means you never get into the `elsif` (if you get into the `else`, you have `!a` so you don't enter the `(els)if`)